src/Symfony/Component/Security/Core/User/UserInterface.php

```diff
@@ -47,7 +47,7 @@
      * and populated in any number of different ways when the user object
      * is created.
      *
-     * @return Role[] The user roles
+     * @return Role[]|string[] The user roles
```

Comment:
@Turneliusz Can you use the PSR-5 notation as it best describes the possible returned values?